stock_quant_sales_channel/models/stock_quant.py

```diff
@@ -7,6 +7,6 @@
 class StockQuant(models.Model):
     _inherit = "stock.quant"
 
-    team_ids = fields.Many2many(
-        related="product_tmpl_id.team_ids", string="Sales Channels",
+    team_id = fields.Many2one(
```

Comment:
I didn't notice that. So, can you please explain me the reason or background why we should have two fields with same field type and same relation? It is not comfortable to me.

Reply:
As you can see from the logic, `team_id` in product_sale_record has the function that if there is a `team_id` in the sale.order, it will be reflected in the product. 
The `team_ids` was a field called sales_candidate. So, this is a field to be entered manually to record as a candidate instead of keeping actual results.

Reply:
So, how about `manual_team_id` with string **Manual Sales Channel**?